docs/gallery/interactive/pyfar_arithmetics.ipynb

```diff
@@ -0,0 +1,237 @@
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "freq_1 = pf.FrequencyData([1, 1, 1], [125, 250, 500])    # single channel\n",
+    "freq_2 = pf.FrequencyData([[1, 1, 1],\n",
+    "                           [2, 2, 2]], [125, 250, 500])  # two channels\n",
+    "\n",
+    "result = freq_1 + freq_2\n",
+    "print(f'result.freq=\\n{result.freq}')"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## DFT normalization and arithmetic operations\n",
+    "\n",
+    "All frequency domain operations on `pyfar.Signal` objects work on `signal.freq_raw`, that is, the signal spectrum without normalization of the Discrete Fourier Transform. The arithmetic operations are implemented in a way that only physically meaningful operations are allowed with respect to the `LINK-TO-FFT-NORMALIZATION`. These rules are motivated by the fact that the normalization denotes if a signal is an energy signal (e.g., an impulse response) or a power signals (e.g., a finite block samples from an otherwise infinite noise process). While addition and subtraction are equivalent in the time and frequency domain, this is not the case for multiplication and division. Nevertheless, **the same rules apply regardless of the domain** for convenience:\n",
+    "\n",
+    "**Addition, subtraction, multiplication, and power**\n",
+    "\n",
+    "* If one signal has the FFT normalization ``'none'``, the results gets the normalization of the other signal.\n",
+    "* If both signals have the same FFT normalization, the results gets the same normalization.\n",
+    "* Other combinations raise an error.\n",
+    "\n",
+    "**Division**\n",
+    "\n",
+    "* If the denominator signal has the FFT normalization ``'none'``, the result gets the normalization of the numerator signal.\n",
+    "* If both signals have the same FFT normalization, the results gets the normalization ``'none'``.\n",
+    "* Other combinations raise an error."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Arithmetic operations using audio objects and arrays\n",
+    "\n",
+    "All arithmetic operations also work between audio objects and scalars and audio objects and arrays if the `shape` of the array can be broadcasted to the `cshape` of the audio object. This means that the same operation is applied to all time or frequency values of an audio object. The example below performes a matrix multiplication between an array of `shape=(4, 3)` and a signal of `cshape(3, )`, which results in a signal of `cshape=(4, )`. Because the same matrix multiplication is done for all time or frequency values, in the example below, it does not matter whether the operation is done in the time or frequency domain. Hence, we directly used the shorthand `@` that is equivalent to using [pf.matrix_multiplication](https://pyfar.readthedocs.io/en/stable/classes/pyfar.audio.html#pyfar.classes.audio.matrix_multiplication) with the default parameters. "
```

Comment:
```suggestion
    "All arithmetic operations also work between audio objects and scalars, and audio objects and arrays if the `shape` of the array can be broadcasted to the `cshape` of the audio object. This means that the same operation is applied to all time or frequency values of an audio object. The example below performes a matrix multiplication between an array of `shape=(4, 3)` and a signal of `cshape(3, )`, which results in a signal of `cshape=(4, )`. Because the same matrix multiplication is done for all time or frequency values, in the example below, it does not matter whether the operation is done in the time or frequency domain. Hence, we directly used the shorthand `@` that is equivalent to using [pf.matrix_multiplication](https://pyfar.readthedocs.io/en/stable/classes/pyfar.audio.html#pyfar.classes.audio.matrix_multiplication) with the default parameters. "
```